client/lib/user-agent-utils/index.js

```diff
@@ -0,0 +1,7 @@
+export function currentUserAgent() {
+	return navigator.userAgent;
+}
```

Comment:
I don't have a problem with providing these tests, though maybe we can make them safe for non-browser use as well.